api/utils.js

```diff
@@ -0,0 +1,18 @@
+const requireAuthentication = (req, res, next) => {
+    const isAuthenticated = req.headers.authorization;
+  
+    if (isAuthenticated) {
+      next();
```

Comment:
This isn't security, this is a stub. I hope theres intentions to actually inspect this header and validate its something meaningful? As of right now, every route is a security vulnerability.